Form/Editors: describe `isDirty`, `clear` and `reset` (#5579)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxForm/1 Configuration/isDirty.md

---
---
##### shortDescription

> Specifies whether editors current values differ from the initial values.

Reviewer: "editors current values" reads wrong to me. First of all, "editor" needs to act as an adjective for the word "value". And it doesn't matter if you talk about a value (singular) or values (plural). The modifier (adjective) stays the same - "editor". You can say "specifies whether the editor value" or "specifies whether editor values". 

Now the word "current" should precede this noun phrase: "current editor values". Otherwise you'd have to switch to possessive - "editors' current values". And it's best to avoid plural possessives. 

Finally, you don't need an article before "initial values". It's plural and not really a specific reference. 

```suggestion
Specifies whether current editor values differ from initial values.
```

##### readonly

---

> This property is a readonly flag. You can use it to check whether the editors in Form were changed. 

Reviewer: ```suggestion
This property is a read-only flag. You can use it to check whether any editor values changed on this Form. 
```

> This property is a read-only flag. You can use it to check whether any editor values changed on this Form. 

Reviewer 2: ```suggestion
This property is a read-only flag. You can use it to check if any editor value changed on this Form. 
```
